src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -149,6 +149,7 @@ public Command ForwardStdOut(TextWriter to = null, bool onlyIfVerbose = false)
                 if (to == null)
                 {
                     _stdOut.ForwardTo(write: Reporter.Output.Write, writeLine: Reporter.Output.WriteLine);
+                    _process.StartInfo.Environment[CommandContext.Variables.AnsiPassThru] = bool.TrueString;
```

Comment:
Alright this seems like a reasonable default, but I think there are testing scenarios where we won't want to have this set.

Can we create a Boolean option which defaults to true for these two methods, and set the value of this environment variable to that Boolean?

That way we will have a method of turning it off if we need to.